lib/bolt/cli.rb

```diff
@@ -425,7 +445,11 @@ def execute(options)
         nodes = executor.from_uris(options[:nodes])
 
         results = nil
+        mode = options[:mode]
+        action = options[:action]
+        object = options[:object]
         outputter.print_head
+        @logger.notice("Starting bolt #{mode} #{action} #{object}")
```

Comment:
This should be logged at info. The user just typed it in we don't need to log it by default